### DISCUSSION TITLE
Failed check of initial species summing to 1

### DISCUSSION THREAD

**Discussion by sjlienge** [AMReX-Combustion/PeleC#790] [Q&A] [answered]
Dear PeleC community,

I would like to run the TGReact case in the RegTests to see how the TGV works in PeleC.
For that I have converted the LiDryer.cti to .yaml format and generated the cantera/profiles.dat file.
When I now want to run the executable I receive the following error:

amrex::Abort::0::Error:: Failed check of initial species summing to 1 !!!

I have tried a few things, e.g. setting all mass fractions defined in the prob.H (line 69-80)
```
#ifdef USE_FUEGO_EOS
      massfrac[H2_ID] =
        prob_parm.d_input[prob_parm.input_H2_init + m * prob_parm.nvars] * f0 +
        prob_parm.d_input[prob_parm.input_H2_init + mp1 * prob_parm.nvars] * f1;
      massfrac[O2_ID] =
        prob_parm.d_input[prob_parm.input_O2_init + m * prob_parm.nvars] * f0 +
        prob_parm.d_input[prob_parm.input_O2_init + mp1 * prob_parm.nvars] * f1;
      massfrac[N2_ID] =
        prob_parm.d_input[prob_parm.input_N2_init + m * prob_parm.nvars] * f0 +
        prob_parm.d_input[prob_parm.input_N2_init + mp1 * prob_parm.nvars] * f1;
      eos.PYT2RE(p, massfrac, T, rho, eint);
#endif
```
to 
```
#ifdef USE_FUEGO_EOS
      massfrac[N2_ID] = 1.0;
      eos.PYT2RE(p, massfrac, T, rho, eint);
#endif
```

I think the problem is very trivial but I dont see it.
Thank you in advance already.

**Comment by marchdf** [comment]
Did you set the other mass fractions to 0 when you set N2 to 1? You need to make sure they are all initialized to something.

**Reply by sjlienge** [reply]
massfrac is initialized as:
`
      amrex::Real massfrac[NUM_SPECIES] = {0.0};
`
I have tried a very blunt:
```
      massfrac[H2O2_ID] = 0.0;
      massfrac[HO2_ID] = 0.0;
      massfrac[OH_ID] = 0.0;
      massfrac[O_ID] = 0.0;
      massfrac[H_ID] = 0.0;
      massfrac[H2O_ID] = 0.0;
      massfrac[O2_ID] = 0.0;
      massfrac[H2_ID] = 0.0;
      massfrac[N2_ID] = 
        1.0 - ( massfrac[H2_ID] + massfrac[O2_ID] + massfrac[H2O_ID] + massfrac[H_ID] 
              + massfrac[O_ID] + massfrac[OH_ID] + massfrac[HO2_ID] + massfrac[H2O2_ID] );
```
But that also does not seem to work :(

**Comment by baperry2** [comment]
Are you compiling with `USE_EXPLICIT_REACT = TRUE` so `Eos_model = Fuego`? If not, that would explain the error you are seeing.

This case is untested and appears to have gotten rather dusty though, so you are likely to encounter some things that need to be modified to get it to run (e.g., it still uses integers rather than strings to specify `pelec.chem_integrator`).

**Reply by baperry2** [reply, marked as answer]
See #793 for the full changes required to get this case to work.

**Reply by sjlienge** [reply]
Thank you so much for looking into this so quickly! I wanted to reproduce the validation from the wiki. The case is running now and I transferred the residual changes from the issue to this setup. I will still be careful with it.
